docs/3.0rc/deploy/infrastructure-examples/serverless.mdx

```diff
@@ -629,14 +629,14 @@ Click **Create** to configure your push work pool by selecting a push option in
  <Tab title="Google Cloud Run">
 
     Each step has several optional fields that are detailed in the 
-    [work pools guide](/3.0rc/deploy/control-runs/work-pools/). 
+    [work pools guide](/3.0rc/deploy/infrastructure-concepts/work-pools/). 
```

Comment:
```suggestion
    [work pools docs](/3.0rc/deploy/infrastructure-concepts/work-pools/). 
```

Reply:
I would use "page" instead.

Reply:
The Google Developer Docs style guide suggests making the link text match the exact text of the title or heading you're referencing. https://developers.google.com/style/link-text#structure-link-text

```suggestion
    [Manage infrastructure with work pools](/3.0rc/deploy/infrastructure-concepts/work-pools/). 
```